### PR TITLE
Implement success vs. failures status codes from symphony.

### PR DESCRIPTION
**Pull request by jkeck**
Begins work on #251 and #423 

Want to get this merged and complete the work and then close the tickets with a subsequent PR.

### CODE REVIEW

File: app/models/symphony_response.rb [hunk 1]
```diff
@@ -10,4 +10,42 @@ def items_by_barcode
...
+  end
+
+  private
+
+  def item_successful?(barcode = nil)
+    return unless barcode && items_by_barcode[barcode]
+    success_codes.include?(items_by_barcode[barcode]['msgcode'])
+  end
+
+  def successfull_barcodes
```
Comment (cbeer) [inline app/models/symphony_response.rb]: `s/successfull_barcodes/successful_barcodes/`

File: app/views/requests/success.html.erb [hunk 2]
```diff
@@ -1,30 +1,40 @@
...
+          <p class='help-block'><%= t('.email_notification.default') %></p>
+        </dd>
+      <% end %>
+    </dl>
+  <% end %>
+
+  <% if symphony_request_failed_due_to_user_privs? && current_request.scannable? && can?(:create, Scan) %>
+    <div class='<%= content_column_class %> <%= label_column_offset_class %>'>
+      <h2>Just need a chapter or article?</h2>
+      <p>Even though your status is blocked, you are eligable for Scan to PDF. You can request a single article or chapter, up to 50 pages.</p>
```
Comment (cbeer) [inline app/views/requests/success.html.erb]: `s/eligable/eligible`

File: app/models/symphony_response.rb (hunk 1 above)
Comment (jkeck) [inline app/models/symphony_response.rb]: :+1: good catch.  Thanks.

File: app/views/requests/success.html.erb (hunk 2 above)
Comment (jkeck) [inline app/views/requests/success.html.erb]: :+1: good catch.  Thanks.

**Pull request merged**